src/blocks/cards-collection/init.php

```diff
@@ -43,10 +53,14 @@ function novablocks_render_cards_collection_block( $attributes, $content ) {
 		ob_start(); ?>
 
 		<div class="<?php echo $className; ?>" style="<?php echo $style; ?>">
-			<?php echo '<' . $titleTag . '>' . $attributes['title'] . '</' . $titleTag . '>'; ?>
-			<p class="intro"><?php echo $attributes['subtitle']; ?></p>
-			<div class="novablocks-cards-collection__layout alignfull">
-				<?php echo $content; ?>
+			<div class="wp-block-group__inner-container">
+				<?php echo '<' . $titleTag . ' class="alignwide">' . $attributes['title'] . '</' . $titleTag . '>'; ?>
+				<p class="intro alignwide"><?php echo $attributes['subtitle']; ?></p>
+				<div class="wp-block alignwide">
```

Comment:
Why did you change from alignfull, to alignwide?

Reply:
Change alignment to "wide" for the block contents to be aligned to the Media Card contents for example. We may add the alignment toolbar in the future if full-aligned is an option.